src/components/messages/MessageCard.js

```diff
@@ -26,7 +27,11 @@ export const MessageCard = ({ message, user, currentUser }) => {
     }
     return (
         <article className="message message--card">
-            <div className="message_text"><em>{user.name}:</em> {message.text}</div>
+            <div className="message_text">
+                <Link to={`/friends/add/${user.id}`}>
+                    <em>{user.name}:</em>
```

Comment:
Tiny request: add a space after the colon in the <em> tags to make it a little more readable?